### ISSUE TITLE
Problem with update

### ISSUE THREAD

**Issue by MarioSWD** [FrSkyRC/ETHOS-Feedback-Community#3762]
Hello everyone,

This is my first post, sorry for my English.

I have a problem with the latest update via ethos suite. As always, I turned on the x20s transmitter with the bootloader turned on in the program, I turned on the update of everything that was old and during the update the transmitter does not want to go through the bootloader update process. The program writes to disconnect the transmitter from the USB and turn it on normally. Unfortunately, the transmitter hangs. The diode is blue, the transmitter cannot be turned on and the screen is still black. If I disconnect the battery and turn it on again, the transmitter cannot be turned on. You can only turn on the bootloader, but then the same thing happens during the update. I even downloaded a new bootloader file, but the program cannot load it - it says that it does not see the transmitter. I don't know what to do at the moment I have a useless transmitter.

Please help.

**Comment by mgsanna** [comment]
Hello Mario, please put the radio in bootloader mode, connect to ethos suite (if you update the suite is better) then update the bootloader to this specific version 1.4.15, (only bootloader) then you can update to whatever version you like

(are you Italian? I'm the translator of Ethos software and of the manual... so you can reply me in Italian, if you like?
<img width="937" alt="Screenshot 2024-03-31 110612" src="https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/15903471/b11cab1a-da72-416f-ae6a-be460dcad79b">

**Comment by strgaltdel** [comment]
the description indicates that the firmware was updated by using an old bootloader,
in this case i'm sure suite is not able to update only the bootloader, because it cant switch to the needed mode furthermore
so you'll have to downgrade firmware to 1.4.x which can be done with tx in bootloader mode
then update bootloader AT FIRST
then busines as usual

https://www.rcgroups.com/forums/showthread.php?4513483-Ethos-1-5-Upgrade-notes-and-typical-issues-after-upgrade

**Comment by MarioSWD** [comment]
thanks for the answers.

I have bootloader version 1.4.3. I did as you wrote, but each time it stays on this screen and nothing happens. The radio screen is black and the blue light is on. Once the USB is disconnected, nothing can be done.

![Zrzut ekranu 2024-03-31 140650](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/165551162/e78afeb7-2856-4d8b-b523-371eaeebef11)


I have to remove the battery from the transmitter to be able to enter bootloader mode again.

**Comment by MarioSWD** [comment]
It worked, thank you very much for your help. I followed step by step as described in this link and it worked:

https://www.rcgroups.com/forums/showpost.php?p=52193291&postcount=1910

Thank you for your help.

**Comment by ibizipizza** [comment]
Hi, I'm having very similar problems on my Twin X-lite. I tried to update everything from 1.4.3 to 1.5.4 but there was a bootloader error and it came to a halt. I tried a few more times but now the Tx won't even turn on - no normal mode or bootloader! When I press the on button, the only indication of life is the 2nd sticky button shows a blue light. 
Without being able to enable bootloader mode, I cant connect to suite to repair it. Any bright ideas would be very happily tried out.
Many thanks
